test/over_react/component_declaration/flux_component_test/store_handlers.dart

```diff
@@ -7,7 +7,7 @@ UiFactory<TestStoreHandlersProps> TestStoreHandlers;
 class TestStoreHandlersProps extends FluxUiProps<TestActions, TestStore> {}
 
 @Component()
-class TestStoreHandlersComponent extends FluxUiComponent<TestStoreHandlersProps> {
+class TestStoreHandlersComponent extends FluxUiComponent {
```

Comment:
This should not have been changed (You should see errors in your analysis server view @clairesarsam-wf  - do you not?)